src/sync/clients.ts

```diff
@@ -92,3 +95,55 @@ export async function setClient(
   clients.set(id, client);
   return setClients(clients, dagWrite);
 }
+
+export async function initClient(
+  dagStore: dag.Store,
```

Comment:
Maybe pass in dag.Write instead? It is not clear yet how this will be used so we will see if this needs its own transaction or not?

Reply:
good point, done.